src/Compilers/CSharp/Test/Symbol/Compilation/SemanticModelAPITests.cs

```diff
@@ -3990,7 +3990,7 @@ static void F()
         }
 
         [WorkItem(976, "https://github.com/dotnet/roslyn/issues/976")]
-        [Fact]
+        [Fact(Skip = "PROTOTYPE(CONSTISTR)")]
```

Comment:
What is failing about this test? Consider recording that in the skip comment.

Reply:
Recorded the reason in the skip comment, is this the correct format?

---
In reply to: [447344626](https://github.com/dotnet/roslyn/pull/45396#discussion_r447344626) [](ancestors = 447344626)

Reply:
That's fine, thanks.

---
In reply to: [447771212](https://github.com/dotnet/roslyn/pull/45396#discussion_r447771212) [](ancestors = 447771212,447344626)